lib/ansible/plugins/action/reboot.py

```diff
@@ -94,18 +94,31 @@ def construct_command(self):
         return reboot_command
 
     def get_system_boot_time(self):
+        stdout = ''
+        stderr = ''
```

Comment:
this might not be py2/py3 compatible, we might want to force bytes

Reply:
Yes, you are correct. One day I'll master that distinction. 😄